keyboards/nix_studio/lilith/info.json

```diff
@@ -0,0 +1,102 @@
+{
+    "manufacturer": "Nix Studio",
+    "keyboard_name": "lilith",
+    "maintainer": "arnstadm",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B13", "B7", "B6", "B5", "B4", "B3", "A15", "A14", "B1", "B0", "A7", "A6", "A5", "A4", "A2", "A3"],
+        "rows": ["A8", "A1", "B12", "B14", "B15"]
+    },
+    "processor": "STM32F411",
+    "url": "https://www.antipode.no/",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x4C49",
+        "vid": "0x6E78"
+    },
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This is `LAYOUT_65_ansi_blocker` community layout?

Reply:
should be fixed now.